frontend/app/src/data/defaults.ts

```diff
@@ -37,5 +38,6 @@ export const exchanges = [
   EXCHANGE_COINBASEPRO,
   EXCHANGE_GEMINI,
   EXCHANGE_CRYPTOCOM,
-  EXCHANGE_BITSTAMP
+  EXCHANGE_BITSTAMP,
+  EXCHANGE_BINANCE_US
```

Comment:
Super small comment, I can probably change myself. Can you put this next to the normal binance? This is where a user would expect to see it when looking for it.